packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -1,6 +1,3 @@
-import React, { useContext, useEffect, useState } from 'react';
```

Comment:
React at the top is idiomatic.

Reply:
oh, it's prettier that auto-formatted it that way I really couldn't care less where the react import lives.